<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_ImprimerFicheCompleteJeu</class>
 <widget class="QDialog" name="F_ImprimerFicheCompleteJeu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Impression de la fiche complête d'un jeu</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Lb_Nom">
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_Nom"/>
     </item>
     <item>
      <widget class="QLabel" name="Lb_Code">
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_Code"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="Lb_Classification">
       <property name="text">
        <string>Classification</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_Classification"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="Lb_AgeMin">
       <property name="text">
        <string>Âge Min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_AgeMin"/>
     </item>
     <item>
      <widget class="QLabel" name="Lb_AgeMax">
       <property name="text">
        <string>Âge Max</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_AgeMax"/>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="Lb_JoueurMin">
       <property name="text">
        <string>Joueur Min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_JoueurMin"/>
     </item>
     <item>
      <widget class="QLabel" name="Lb_JoueurMax">
       <property name="text">
        <string>Joueur Max</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_JoueurMax"/>
     </item>
    </layout>
   </item>
   <item row="3" column="1" rowspan="10">
    <widget class="QLabel" name="Lb_Photo">
     <property name="maximumSize">
      <size>
       <width>470</width>
       <height>303</height>
      </size>
     </property>
     <property name="text">
      <string>Photo</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="Lb_Createur">
       <property name="text">
        <string>Créateur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_Createur"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="Lb_Editeur">
       <property name="text">
        <string>Editeur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_Editeur"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QLabel" name="Lb_Fournisseur">
       <property name="text">
        <string>Fournisseur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_Fournisseur"/>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="Lb_Statut">
       <property name="text">
        <string>Statut</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_Statut"/>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="Lb_Etat">
       <property name="text">
        <string>Etat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_Etat"/>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="Lb_EtatInitial">
       <property name="text">
        <string>Etat Initial</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_EtatInitial"/>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="Lb_DateAchat">
       <property name="text">
        <string>Date Achat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_DateAchat"/>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="Lb_PrixAchat">
       <property name="text">
        <string>Prix Achat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_PrixAchat"/>
     </item>
    </layout>
   </item>
   <item row="12" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="Lb_PrixLoc">
       <property name="text">
        <string>Prix Loc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_PrixLoc"/>
     </item>
    </layout>
   </item>
   <item row="13" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="Lb_Caution">
       <property name="text">
        <string>Caution</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_Caution"/>
     </item>
    </layout>
   </item>
   <item row="14" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="Lb_Emplacement">
       <property name="text">
        <string>Emplacement</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_Emplacement"/>
     </item>
    </layout>
   </item>
   <item row="15" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="Lb_Remarques">
         <property name="text">
          <string>Remarques</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="TxE_Remarques"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="Lb_Description">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="TxE_Description"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="Lb_Contenu">
         <property name="text">
          <string>Contenu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="TxE_Contenu"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="16" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QCommandLinkButton" name="Bt_Imprimer">
       <property name="text">
        <string>Imprimer cette fiche</string>
       </property>
       <property name="icon">
        <iconset resource="ImagesIcones.qrc">
         <normaloff>:/Valider.png</normaloff>:/Valider.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="Bt_Quitter">
       <property name="text">
        <string>Retour au jeu</string>
       </property>
       <property name="icon">
        <iconset resource="ImagesIcones.qrc">
         <normaloff>:/Annuler.png</normaloff>:/Annuler.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ImagesIcones.qrc"/>
 </resources>
 <connections/>
</ui>
